packages/gateway/src/graphql/schema.d.ts

```diff
@@ -219,6 +222,19 @@ export interface GQLRegistrationCountResult {
   total: number
 }
 
+export type GQLRecordDetails = GQLBirthRegistration | GQLDeathRegistration
+
+/** Use this to resolve union type RecordDetails */
```

Comment:
Notice that this file gets automatically regenerated whenever you change `packages/gateway/src/features/registration/schema.graphql`